### ISSUE TITLE
生成的VLESS+TLS+WS[仅CDN推荐] VMess+TLS+WS[仅CDN推荐无法使用任何优选IP和域名

### ISSUE THREAD

**Issue by Neomanbeta** [mack-a/v2ray-agent#1088]
不按照模版提交，一律关闭，反馈之前请仔细阅读[脚本异常处理](https://www.v2ray-agent.com/archives/1684115970026)，[客户端使用教程](https://www.v2ray-agent.com/archives/1695534611317)、[八合一脚本从入门到精通](https://www.v2ray-agent.com/archives/1710141233)。

如果低版本升级高版本出现问题，建议使用**20.卸载脚本**卸载后重新安装

## 1.问题描述
- 尽可能的将复现步骤描述清楚
```
由于v3.3.14和v3.3.16无论是升级安装和卸载后重新安装都没有发生任何错误，安装的时候使用的是自定义安装，只安装1,3,6,7,9。生成的节点测试也都全部可以正常使用。但是VLESS+TLS+WS和VMess+TLS+WS不能像之前一样使用优选CDN，无论是从脚本中选10CDN管理去生成，还是手动修改服务器地址和端口，比如使用visa.cn端口2087，都无法连接。cloudflare上的TLS已经设置成FULL。
```

## 2.安装失败的log或者截图
```
测试错误: Get "http://www.gstatic.com/generate_204": unexpected HTTP response status: 521
```

## 3.系统版本
```
Debian12
```

## 4.脚本版本
```
v3.3.16
```

## 5.服务端安装的内核、协议、安装方式
sing-box
```
请在这里填写
```
- 安装方式
```
任意组合安装 1,3,6,7,9
```  
- 协议(0.VLESS+TLS_Vision+TCP)
```
VLESS+TLS+WS和VMess+TLS+WS
```  
## 6.客户端版本
```
# 
Nekobox 4.0 beta3和Karing 1.0.28.371
```

**Comment by mack-a** [comment]
检查nginx是否正常启动。

**Comment by Neomanbeta** [comment]
root@OC2:~# /etc/init.d/nginx status
● nginx.service - nginx - high performance web server
     Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2024-09-03 22:34:02 CST; 1h 6min ago
       Docs: https://nginx.org/en/docs/
    Process: 115861 ExecStart=/usr/sbin/nginx -c /etc/nginx/nginx.conf (code=exited, status=0/SUCCESS)
   Main PID: 115862 (nginx)
      Tasks: 3 (limit: 1125)
     Memory: 2.8M
        CPU: 15ms
     CGroup: /system.slice/nginx.service
             ├─115862 nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf
             ├─115863 nginx: worker process
             └─115864 nginx: worker process
但是伪装站打不开。
btw:说个和这个无关的，读取上次配置的时候，读取vless和vmess的path，vmess的path也会读取成vless的。

**Comment by mack-a** [comment]
服务端有没有日志产生，检查下核心是否正常启动。

**Comment by Neomanbeta** [comment]
root@OC2:~# cat /var/log/nginx/error.log
2024/09/04 13:18:21 [notice] 115862#115862: signal 15 (SIGTERM) received from 139651, exiting
2024/09/04 13:18:21 [notice] 115864#115864: exiting
2024/09/04 13:18:21 [notice] 115864#115864: exit
2024/09/04 13:18:21 [notice] 115863#115863: exiting
2024/09/04 13:18:21 [notice] 115863#115863: exit
2024/09/04 13:18:21 [notice] 115862#115862: signal 15 (SIGTERM) received from 1, exiting
2024/09/04 13:18:21 [notice] 115862#115862: signal 17 (SIGCHLD) received from 115864
2024/09/04 13:18:21 [notice] 115862#115862: worker process 115864 exited with code 0
2024/09/04 13:18:21 [notice] 115862#115862: signal 29 (SIGIO) received
2024/09/04 13:18:21 [notice] 115862#115862: signal 17 (SIGCHLD) received from 115863
2024/09/04 13:18:21 [notice] 115862#115862: worker process 115863 exited with code 0
2024/09/04 13:18:21 [notice] 115862#115862: exit
2024/09/04 13:18:21 [notice] 139652#139652: using the "epoll" event method
2024/09/04 13:18:21 [notice] 139652#139652: nginx/1.25.5
2024/09/04 13:18:21 [notice] 139652#139652: built by gcc 10.2.1 20210110 (Debian 10.2.1-6) 
2024/09/04 13:18:21 [notice] 139652#139652: OS: Linux 6.7.9-bbrplus
2024/09/04 13:18:21 [notice] 139652#139652: getrlimit(RLIMIT_NOFILE): 1024:524288
2024/09/04 13:18:21 [notice] 139655#139655: start worker processes
2024/09/04 13:18:21 [notice] 139655#139655: start worker process 139656
2024/09/04 13:18:21 [notice] 139655#139655: start worker process 139657

核心是否启动指的是nginx么？如果是，nginx查看运行是正常的，进程里也存在
root@OC2:~# ps aux | grep nginx
root      139655  0.0  0.1   9948  1520 ?        Ss   13:18   0:00 nginx: master process /usr/sbin/nginx -c /etc/nginx/nginx.conf
nginx     139656  0.0  0.3  10656  3440 ?        S    13:18   0:00 nginx: worker process
nginx     139657  0.0  0.3  10656  3440 ?        S    13:18   0:00 nginx: worker process
root      139870  0.0  0.2   6240  2304 pts/0    S+   13:28   0:00 grep nginx

root@OC2:~# nginx -t
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful

**Comment by mack-a** [comment]
你用的sing-box的话检查一下是否正常启动。如果是xray则检查xray。

**Comment by Neomanbeta** [comment]
我用的就是sing-box核心，sing-box是没问题的，正常运行，因为所有节点都正常使用。即使那两个CDN节点，用默认生成的域名和端口也是正常使用的。

**Comment by mack-a** [comment]
ws的端口是否使用以下端口。
```
443
2053
2083
2087
2096
8443
```

**Comment by Neomanbeta** [comment]
![test](https://github.com/user-attachments/assets/039d5167-a6cf-4d18-b436-baa5bea21dd3)
freeyx.cloudflare88.eu.org是优选域名里面都是CF官方IP，上面neomanoc2-VLESS_WS是我八合一脚本创建的CDN节点。下面 韩国首尔 那个其它服务器我用来对照的节点。

**Comment by mack-a** [comment]
你访问ws节点，服务端singbox核心有没有日志产生？

**Comment by Neomanbeta** [comment]
root@OC1:~# cd /etc/v2ray-agent/sing-box/
root@OC1:/etc/v2ray-agent/sing-box# ls
conf  sing-box
root@OC1:/etc/v2ray-agent/sing-box# cd ..
root@OC1:/etc/v2ray-agent# ls
backup_crontab.cron  cdn  crontab_tls.log  install.log	install.sh  nginx_error.log  sing-box  subscribe  subscribe_local  subscribe_remote  tls  v2ray  warp  xray
root@OC1:/etc/v2ray-agent# 

如果我没找错目录的话，并没有看到sing-box产生任何log

**Comment by mack-a** [comment]
vasma->16->2->6

**Comment by Neomanbeta** [comment]
+0800 2024-09-04 16:00:54 INFO sing-box started (0.00s)
+0800 2024-09-04 16:01:14 INFO [4064299948 0ms] inbound/vless[VLESSWS]: inbound connection from 115.171.37.139:15130
+0800 2024-09-04 16:01:14 INFO [1608468806 97ms] inbound/vless[VLESSWS]: [neomanoc2-VLESS_WS] inbound connection to www.gstatic.com:80
+0800 2024-09-04 16:01:14 INFO [1608468806 97ms] outbound/direct: outbound connection to www.gstatic.com:80
+0800 2024-09-04 16:01:14 DEBUG [1608468806 97ms] dns: lookup domain www.gstatic.com
+0800 2024-09-04 16:01:14 DEBUG [1608468806 99ms] dns: lookup succeed for www.gstatic.com: 142.251.42.131 2404:6800:4004:825::2003
+0800 2024-09-04 16:01:17 DEBUG [4064299948 3.9s] inbound/vless[VLESSWS]: connection closed: process connection from 115.171.37.139:15130: upload: ws closed: 1000  | download: raw-read tcp 10.0.0.4:55730->142.251.42.131:80: use of closed network connection
+0800 2024-09-04 16:03:04 INFO [680697071 0ms] inbound/vless[VLESSWS]: inbound connection from 115.171.37.139:15204
+0800 2024-09-04 16:03:05 INFO [3953627952 1.8s] inbound/vless[VLESSWS]: [neomanoc2-VLESS_WS] inbound connection to www.gstatic.com:80
+0800 2024-09-04 16:03:05 INFO [3953627952 1.8s] outbound/direct: outbound connection to www.gstatic.com:80
+0800 2024-09-04 16:03:08 DEBUG [680697071 4.8s] inbound/vless[VLESSWS]: connection closed: process connection from 115.171.37.139:15204: download: raw-read tcp 10.0.0.4:45920->142.251.42.131:80: use of closed network connection | upload: ws closed: 1000

**Comment by mack-a** [comment]
看的没问题
